doc/src/conf.py

```diff
@@ -81,6 +81,8 @@
     "tutorial/matrices": "../tutorials/intro-tutorial/matrices.html",
     "tutorial/manipulation": "../tutorials/intro-tutorial/manipulation.html",
 
+    "tutorials/biomechanics": "tutorials/physics/biomechanics",
```

Comment:
You should refer to the html pages here, e.g. `"../tutorials/physics/biomechanics/biomechanics.html"`.
Also, make sure to have references in the index files (`index.rst`), e.g. a reference to a tutorials/physics/index.rst file in [`tutorials/index.rst`](https://github.com/sympy/sympy/blob/master/doc/src/tutorials/index.rst).

Reply:
No need to make a redirect for anything that has not yet been released. This stuff is not in SymPy 1.12.